app-modules/division/src/Filament/Resources/DivisionResource/Pages/ViewDivision.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Assist\Division\Filament\Resources\DivisionResource\Pages;
+
+use Filament\Actions\EditAction;
+use Filament\Infolists\Infolist;
+use Assist\Division\Models\Division;
+use App\Filament\Resources\UserResource;
+use Filament\Resources\Pages\ViewRecord;
+use Filament\Infolists\Components\Section;
+use Filament\Infolists\Components\TextEntry;
+use Assist\Division\Filament\Resources\DivisionResource;
+
+class ViewDivision extends ViewRecord
+{
+    protected static string $resource = DivisionResource::class;
+
+    public function infolist(Infolist $infolist): Infolist
+    {
+        return parent::infolist($infolist)
+            ->schema([
+                Section::make()
+                    ->schema([
+                        TextEntry::make('id')
+                            ->label('ID'),
+                        TextEntry::make('name'),
+                        TextEntry::make('code'),
+                        TextEntry::make('header'),
+                        TextEntry::make('footer'),
```

Comment:
You may want to output these much in the same way that we do for the other tiptap fields in KnowledgeBase